src/custom/pages/Swap/index.tsx

```diff
@@ -375,6 +386,132 @@ const SwapButton = ({ children, showLoading, showButton = false }: SwapButtonPro
     </Text>
   )
 
+interface HighFeeContainerProps {
+  padding?: string
+  margin?: string
+  width?: string
+}
+
+// const ShowMoreButton = styled.span`
+//   text-decoration: underline;
+//   cursor: pointer;
+//   margin-left: auto;
+// `
+
+const WarningCheckboxContainer = styled.small`
+  &&&& {
+    display: flex;
+    font-weight: bold;
+    gap: 2px;
+    justify-content: center;
+    align-items: center;
+  }
+`
+
+const HighFeeWarningContainer = styled(AuxInformationContainer).attrs((props) => ({
+  ...props,
+  hideInput: true,
+}))<HighFeeContainerProps>`
+  padding: ${({ padding = '5px 16px' }) => padding};
+  font-size: small;
+  font-weight: 700;
+  &&&&& {
+    background: ${({ theme }) => transparentize(0.8, theme.red1)};
+    color: ${({ theme }) => theme.red1};
+  }
+  width: ${({ width = '100%' }) => width};
+  border-radius: ${({ theme }) => theme.buttonPrimary.borderRadius};
+  margin: ${({ margin = '0 auto 12px auto' }) => margin};
+  > small {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 2px;
+    svg {
+      &:first-child {
+        margin-right: 4px;
+      }
+      stroke: ${({ theme }) => theme.red1};
+    }
+
+    > div:first-of-type {
+      margin-top: 2px;
+    }
+    > ${WarningCheckboxContainer} {
+      margin-left: auto;
+      > input {
+        cursor: pointer;
+      }
+    }
+  }
+  > div {
+    display: flex;
+    padding: 8px 0;
+  }
+`
+
+const ErrorStyledInfo = styled(StyledInfo)`
+  color: ${({ theme }) => theme.red1};
+`
+
+const HighFeeWarningMessage = ({ feePercentage }: { feePercentage?: Fraction }) => (
+  <div>
+    <small>
+      Current fees on this network make up{' '}
+      <u>
+        <strong>{feePercentage?.toFixed(2)}%</strong>
+      </u>{' '}
+      of your input swap amount.
+      <br />
+      You may still move forward with this swap but it is highly recommended that you wait for lower fees, or switch to
+      xDai.
+    </small>
+  </div>
+)
+
+export type HighFeeWarningProps = { trade?: TradeGp; acceptWarningCb?: () => void } & HighFeeContainerProps
+
+const HIGH_FEE_DEBOUNCE_TIME = 400
+
+export const HighFeeWarning = (props: HighFeeWarningProps) => {
+  const { trade, acceptWarningCb } = props
+
+  const theme = useContext(ThemeContext)
+
+  // only considers inputAmount vs fee (fee is in input token)
+  const [isHighFeeRaw, feePercentage] = useMemo(() => {
+    if (!trade) return []
+
+    const feePercentage = trade.fee.feeAsCurrency.divide(trade.inputAmount).asFraction
+    return [feePercentage.greaterThan(FEE_SIZE_THRESHOLD), feePercentage.multiply('100')]
+  }, [trade])
+
+  // debounce the calcublation of high fee boolean
+  const isHighFee = useDebounce(!!isHighFeeRaw, HIGH_FEE_DEBOUNCE_TIME)
+
+  if (!isHighFee) return null
+
+  return (
+    <HighFeeWarningContainer {...props}>
+      <small>
```

Comment:
One of the sub elements/components is a rendered `<div>`. While Chrome renders `<div>`'s in a nested `<small>` element, this apparently is not valid HTML. Replace the `<small>` element here with a `<div>` instead.